shared/utility/math/angle.h

```diff
@@ -85,6 +86,10 @@ namespace math {
             return d;
         }
 
+        inline double vectorToBearing(const Eigen::Vector2d& dirVec) {
+            return std::atan2(dirVec.y(), dirVec.x());
+        }
```

Comment:
Really?...

Reply:
Not sure what you are asking here

Reply:
Not sure what you are asking here

Reply:
That indeed was a terrible statement with no context.
What I was meaning is that this function just calls another function. There is no complexity being masked so I think this function is unecessary. Whoever is calling it could just use atan2 themselves